README.md

```diff
@@ -42,7 +42,13 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+
+for (i = 0; i < foods.length; i++)
+  for (j = 0;j <= foods[0].length; j++) {
+console.log('I want to eat a ' + foods[i][j])
+}
+
```

Comment:
close! you can actually do this without the first for loop you added in there. Right now, your loop will actually print out the fruits twice! instead you would just run 
```
for (j = 0;j < foods[0].length; j++) {
 +console.log('I want to eat a ' + foods[i][j])
 +}
```
You'll also want to make sure in this case that you're only doing <, not <=. By including the = it will actually make one extra loop and print undefined